internal/resources/metal/organization_member/models.go

```diff
@@ -0,0 +1,47 @@
+package organizationmember
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+
+	"github.com/packethost/packngo"
+)
+
+type ResourceModel struct {
+	ID             types.String `tfsdk:"id"`
+	Invitee        types.String `tfsdk:"invitee"`
+	Invited_by     types.String `tfsdk:"invited_by"`
```

Comment:
The name of the field should follow Go naming patterns:
```suggestion
	InvitedBy     types.String `tfsdk:"invited_by"`
```